go.mod

```diff
@@ -16,7 +16,7 @@ require (
 	github.com/ipfs/go-datastore v0.4.4
 	github.com/ipfs/go-log v1.0.4
 	github.com/keep-network/go-libp2p-bootstrap v0.0.0-20200423153828-ed815bc50aec
-	github.com/keep-network/keep-common v1.4.1-0.20210319095805-ebf46d0b62db
+	github.com/keep-network/keep-common v1.4.1-0.20210426115544-f1c11be35fce
```

Comment:
Can we update this version to the latest  merge commit (`3f1aa45d984ac25a1c5b4b1f511a790f69817072`)?

Reply:
Updated, see 1e06979505d9024b05fa6ebe1f77bce74ec0a01c.